### ISSUE TITLE
Adopted child, add copy to submit additional information

### ISSUE THREAD

**Issue by vbellissimo** [department-of-veterans-affairs/va.gov-team#76299]
## Issue Description
As a veteran with an adopted child,
I need to understand/see if I am required to submit any additional information related to this,
so that I can have my Pension application processes properly.

As a Pension team,
we need to ensure we let the a veteran with an adopted child know about any required additional information they need to submit with their online application,
so that the business (VBA) can process their application properly.

Supporting information required based on the PDF form:

<img width="1135" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOUZjQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--a46cea4cbc1c59aa5b73ac46695bb6e12c3be265/Screenshot%20%28419%29.png" alt="Screenshot (419).png" />

Current question asking about child relationship related to adoption:

<img width="1820" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOXhjQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--fe9d371c0a9c23b9417cc860572382b5ab8762df/Screenshot%20%28418%29.png" alt="Screenshot (418).png" />

---
## Tasks

- [ ] Design to identify a solution on how to incorporate an alert that lets the the veteran know that if their child is adopted,  they need to submit adoption papers or an amended birth certificate. Some areas to consider placing this information could be:
-- the introduction page
-- the page where the child relationship question exists; and/or
-- the supporting documents page toward the end of the form (using conditional copy) 
- [ ] These supporting documents will be able to be uploaded or they can be mailed in to the Pension address provided toward the end of the form.
- [ ] Create engineer ticket to implement the changes or consult with Anh/Product Mgr to create ticket

## Acceptance Criteria
- Any veteran with an adopted child who is applying for Pension benefits online will know they need to submit adoption papers or an amended birth certificate as part of the application process.

**Comment by aplatt-coforma** [comment]
@vbellissimo - I removed the frontend and engineering labels on this ticket. I think we can make this the design one first then make a new ticket for engineering once Gretchen is done with the designs.

**Comment by gmaciolek** [comment]
I've updated the [Figma design for the dependent children information page to include this information in an alert](https://www.figma.com/file/9JKK5Eo43uJWEr66JPiebc/Pension-Claim-Form-21P-527EZ?type=design&node-id=2292%3A32130&mode=design&t=WbyVpMlv8GEPKtKt-1) The alert should display as shown if the "They're my adopted child" radio button is selected. 

This follows the pattern of how we currently communicate to users when additional documentation is required. 

I don't like this solution for various reasons (accessibility concerns, feedback from the governance team, general best practices, anecdotal observations in usability testing), but I'm guessing it's higher priority to get this information up quickly vs. stopping to re-think our approach to requesting additional documentation.

**Comment by gmaciolek** [comment]
Julie Strothman approved of this solution for a first pass at adding this info. She also requested the following changes to the supporting documentation page:

- If the user selected “They’re my adopted child” for any dependent, the documents list includes “Adoption papers or an amended birth certificate for your adopted child or children”
- Update the bulleted lists of documents to all start with a capital letter -- I think they currently do not.
